WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -77,6 +77,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
```

Comment:
I think I differ from Jeremy in that I pretty strongly prefer `unless` to `if !`. But I always prefer `if` to `unless` otherwise. No `unless ... else`, for example.

(sorry to spam the review)